yb-voyager/cmd/exportData.go

```diff
@@ -241,6 +241,23 @@ func exportData() bool {
 
 	// Check if source DB has required permissions for export data
 	if source.RunGuardrailsChecks {
```

Comment:
Move all this logic to a separate function? It's getting too long to include in the `exportData` function itself.